### DISCUSSION TITLE
Help with downloading data

### DISCUSSION THREAD

**Discussion by Ira-marriott** [AdaptiveMotorControlLab/CEBRA#75] [Q&A] [answered]
Hi, 

I am really struggling with using the functions in the demo that install the data. I followed the steps for installing everything and then in the first demo when i get to hippocampus_pos = cebra.datasets.init('rat-hippocampus-single-achilles') i get an error FileNotFoundError: [Errno 2] No such file or directory

Sorry if this is super basic - I'm relatively new to python 

Really appreciate any help

Thanks,
Ira

**Comment by MMathisLab** [comment, marked as answer]
No problem! Actually the dataloader is now built into the code, but not in the stable release - sorry for any confusion! You can try the notebooks as is with pip install cebra==0.3.0rc1

**Reply by Ira-marriott** [reply]
Thanks!
